### PR TITLE
Improve generated file name in both OpenAPI commands

### LINKED ISSUE #137: Improve error messages
Will list issues I find in error messages related to the `openapi` command here. 
```
bal openapi -i petstore.yml
Note: This is an experimental tool, which only supports a limited set of functionality.
error: An OpenApi definition file is required to generate the service.
E.g: ballerina openapi --input <OpenApiContract> or <Ballerina file>
```
- Do we need to keep printing that experimental tool warning each time we run the `openapi` command? 
- Typos in the error message: 
OpenApi -> OpenAPI
E.g.: -> e.g., 
ballerina -> bal

### PR DESCRIPTION
**Pull request by lnash94**
## Purpose
Fix for improving error messages, the behaviour of the file regenerate prompt,  and formatting issues with generated schema file.
Update the dependency versions.
> Fix 
#137
#138 
         #139
         #151

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java [hunk 1]
```diff
@@ -293,7 +293,7 @@ private static String checkAvailabilityOfGivenName(String openApiName, File[] li
         for (File file : listFiles) {
             if (file.getName().equals(openApiName)) {
                 String userInput = System.console().readLine("There is already a/an " + file.getName() +
-                        " in the location. Do you want to override the file [Y/N]? ");
+                        " in the location. Do you want to override the file [N]? ");
```
Comment (NipunaRanasinghe) [inline openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java:296]: Why have we removed `[Y]es` option in here?

File: openapi-cli/src/main/java/org/ballerinalang/generators/BallerinaSchemaGenerator.java [hunk 2]
```diff
@@ -149,7 +150,7 @@ public static SyntaxTree generateSyntaxTree(Path definitionPath)
                             Token openSBracketToken = AbstractNodeFactory.createIdentifierToken("[");
                             Token closeSBracketToken = AbstractNodeFactory.createIdentifierToken("]");
                             IdentifierToken fieldName =
-                                    AbstractNodeFactory.createIdentifierToken(" " + escapeIdentifier(
+                                    AbstractNodeFactory.createIdentifierToken("" + escapeIdentifier(
```
Comment (NipunaRanasinghe) [inline openapi-cli/src/main/java/org/ballerinalang/generators/BallerinaSchemaGenerator.java:153]: Cant we simplify this as follows? 
```ballerina
IdentifierToken fieldName = AbstractNodeFactory.createIdentifierToken(escapeIdentifier(
                                            schema.getKey().trim().toLowerCase(Locale.ENGLISH)) + "list");
```

File: openapi-cli/src/main/java/org/ballerinalang/generators/BallerinaSchemaGenerator.java [hunk 3]
```diff
@@ -205,7 +206,7 @@ private static void addRecordFields(List<String> required, List<Node> recordFiel
         RecordFieldNode recordFieldNode;
         //FiledName
         IdentifierToken fieldName =
-                AbstractNodeFactory.createIdentifierToken(" " + escapeIdentifier(field.getKey().trim()));
+                AbstractNodeFactory.createIdentifierToken("" + escapeIdentifier(field.getKey().trim()));
```
Comment (NipunaRanasinghe) [inline openapi-cli/src/main/java/org/ballerinalang/generators/BallerinaSchemaGenerator.java:209]:
```suggestion
                AbstractNodeFactory.createIdentifierToken(escapeIdentifier(field.getKey().trim()));
```

File: openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java (hunk 1 above)
Comment (lnash94) [reply openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java:296]: Capitalize represents the default/recommended option. I removed this sort of prompts due to avoiding user confusion in between default value[Yes and No]. Could you please suggest some suitable way to represent these two values? fixed for this issue [138](https://github.com/ballerina-platform/ballerina-openapi/issues/138)

File: openapi-cli/src/main/java/org/ballerinalang/generators/BallerinaSchemaGenerator.java (hunk 2 above)
Comment (lnash94) [reply openapi-cli/src/main/java/org/ballerinalang/generators/BallerinaSchemaGenerator.java]: yeah we can, I will simplify.

File: openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java (hunk 1 above)
Comment (NipunaRanasinghe) [reply openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java:296]: Yeah I agree. But capitalizing the default option does not meaning removing the alternative IMO. How about using `Do you want to override the file? (y/N) `?

File: openapi-cli/src/main/java/org/ballerinalang/openapi/CodeGenerator.java [hunk 4]
```diff
@@ -319,7 +319,7 @@ private void writeGeneratedSources(List<GenSrcFile> sources, Path srcPath, Path
                     for (GenSrcFile gFile : sources) {
                         if (file.getName().equals(gFile.getFileName())) {
                             String userInput = System.console().readLine("There is already a/an " + file.getName() +
-                                    " in the location. Do you want to override the file [Y/N]? ");
+                                    " in the location. Do you want to override the file [N]? ");
```
Comment (NipunaRanasinghe) [inline openapi-cli/src/main/java/org/ballerinalang/openapi/CodeGenerator.java:322]: https://github.com/ballerina-platform/ballerina-openapi/pull/154#discussion_r602356452

File: openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java (hunk 1 above)
Comment (lnash94) [reply openapi-cli/src/main/java/org/ballerinalang/ballerina/OpenApiConverterUtils.java:296]: Yeah, this way looks good to me. User can explicitly get the default value here. I will change with this.

**Review by NipunaRanasinghe — APPROVED** [review]

File: openapi-cli/src/main/java/org/ballerinalang/openapi/OpenApiMesseges.java [hunk 5]
```diff
@@ -26,49 +26,12 @@
...
-    public static final String MODULE_MD_EXCEPTION = "Unable to create moudle.md file. File system error occured.";
-    public static final String DEFINITION_EXISTS = "There is already an OpenApi contract in the location.";
     public static final String EXPERIMENTAL_FEATURE = "Note: This is an experimental tool, which only" +
             " supports a limited set of functionality.";
-    public static final String MESSAGE_FOR_MISSING_INPUT = "An OpenApi definition file is required to generate the " +
-            "service. \nE.g: ballerina openapi --input <OpenApiContract> or <Ballerina file>";
-    public static final String MESSAGE_FOR_INVALID_MODULE = "The module provided is not found in the " +
-            "current location.";
+    public static final String MESSAGE_FOR_MISSING_INPUT = "An OpenAPI definition file is required to generate the " +
+            "service. \ne.g: bal OpenAPI --input <OpenAPIContract> or <Ballerina file>";
```
Comment (pubudu91) [inline openapi-cli/src/main/java/org/ballerinalang/openapi/OpenApiMesseges.java:34]: e.g: => e.g.,

**Pull request merged**